src/aaudio/AAudioLoader.cpp

```diff
@@ -38,15 +38,18 @@ int AAudioLoader::open() {
     if (mLibHandle != nullptr) {
         return 0;
     }
-    mLibHandle = dlopen(LIB_AAUDIO_NAME, 0);
+    mLibHandle = dlopen(LIB_AAUDIO_NAME, 0);  // RTLD_LAZY instead of 0?
```

Comment:
I agree that this should not be 0. I am concerned about real-time performance. With RTLD_LAZY, will the first call be slower? I load all the functions. Maybe RTLD_NOW would be better.

Reply:
Sorry, I might've been mistaken, but I thought that `RTLD_LAZY` is `0`, so my point was to use explicit identified instead of a literal value.